exampleplugins/5.2/SwagExtendCustomer/Resources/views/backend/swag_extend_customer/view/detail/window.js

```diff
@@ -0,0 +1,67 @@
+//{block name="backend/customer/view/detail/window"}
+// {$smarty.block.parent}
+Ext.define('Shopware.apps.SwagExtendCustomer.view.detail.Window', {
+    override: 'Shopware.apps.Customer.view.detail.Window',
+
+    getTabs: function () {
+        var me = this,
+            result = me.callParent();
+
+        result.push(Ext.create('Shopware.apps.SwagExtendCustomer.view.detail.MyOwnTab'));
+
+        return result;
+    },
+
+    /**
+     * Replace the textBox for field "title" with a comboBox
+     *
+     * @returns { Ext.form.FieldSet }
+     */
+    createPersonalFieldSet: function () {
+        var me = this,
+            fieldSet = me.callParent(arguments);
+
+        fieldSet.items.items[1].items.items[0] = Ext.create('Ext.form.field.ComboBox', {
```

Comment:
do you overwrite the existing salutation field? maybe we should identify it in another way, so it is safe if other plugins do a similar extension